features/networking/sdn.feature

```diff
@@ -412,3 +412,18 @@ Feature: SDN related networking scenarios
       | since         | 30s                            |
     And the output should contain:
       | Not applying unsafe change: invalid configuration |
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-36287
+  @admin
+  @destructive
+  Scenario: Netnamespace should be recreated after deleting it before the project is deleted
+    Given the env is using "OpenShiftSDN" networkType
+    Given I have a project
+    And admin checks that the "<%= project.name %>" net_namespace exists
```

Comment:
Is admin here (and below) a must ? or we are OK to use `And I check that ...` ?

Reply:
yes, admin is a must since the netnamespace is cluster scope.